community/learn/hasura/graphql-tutorial-backend/tutorial-site/content/custom-business-logic.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Custom Business Logic"
+---
+
+Hasura gives you CRUD + realtime GraphQL APIs with authorization & access control. However there are cases where you would want to query something which is not in your database.
```

Comment:
Second line should say: However, there are case where you would want to add custom/business logic in your app. Some examples are ....